torch/csrc/jit/passes/memory_planning/linear_scan.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include <torch/csrc/jit/passes/memory_planning.h>
+#include <torch/csrc/jit/runtime/static/impl.h>
+
+namespace torch {
+namespace jit {
+
+using EndSortedLiveRangeMap =
```

Comment:
it's only used in the `cpp` file, why don't we move it there?

Reply:
i can't remember if it's used in any of the other diffs but it's a vaguely useful typedef? not sure. another coin flip.